README.md

```diff
@@ -12,24 +12,23 @@
 
 ## Project
 
-Wasabi A/B Testing Service is a real-time, enterprise-grade, 100% API driven project. Users are empowered to own their own data, and run experiments across web, mobile, and desktop. It’s fast, easy to use, it’s chock full of features, and instrumentation is minimal.
-
-Learn more about how Wasabi can empower your team to move from hunches to actionable, data-driven user insights with our simple, flexible, and scalable experimentation platform.
+Wasabi is a mature, feature-rich, open-source A/B testing platform, complete with an experiment management UI. Wasabi powers all of Intuit's in-product experimentation and has been running in production at scale for several years.
```

Comment:
@toddfast Evaluators might be confused that this platform is Intuit-specific when they see this in the 1st paragraph: "Wasabi powers all of Intuit's in-product experimentation and has been running in production at scale for several years" . Would it make sense to move it to the 2nd paragraph as a proof of scale? [cc: @calvinwma @asidhwani1]